app-modules/report/src/Filament/Widgets/UserUniqueLoginCountLineChart.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\Report\Filament\Widgets;
+
+use App\Models\User;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Cache;
+
+class UserUniqueLoginCountLineChart extends LineChartReportWidget
+{
+    protected static ?string $heading = 'Unique Logins By Month';
+
+    protected int | string | array $columnSpan = [
+        'sm' => 1,
+        'md' => 4,
+        'lg' => 4,
+    ];
+
+    protected function getOptions(): array
+    {
+        return [
+            'plugins' => [
+                'legend' => [
+                    'display' => false,
+                ],
+            ],
+            'scales' => [
+                'y' => [
+                    'min' => 0,
+                ],
+            ],
+        ];
+    }
+
+    protected function getData(): array
+    {
+        $runningTotalPerMonth = Cache::tags([$this->cacheTag])->remember('user-unique-login-count-line-chart', now()->addHours(24), function (): array {
+            $totalCreatedPerMonth = User::query()
+                ->selectRaw('date_trunc(\'month\', created_at) as month')
+                ->selectRaw('count(*) as total')
+                ->with('logins')
+                ->groupBy('month')
+                ->orderBy('month')
+                ->pluck('total', 'month');
```

Comment:
Are you sure this is getting the correct data? This kind of looks like it is getting the amount of Users created per month.. Not login events like it is supposed to.

I think you should be doing this based on the tracked event of the user login type.